src/applications/personalization/profile/tests/components/direct-deposit/BankInfoCNP.unit.spec.jsx

```diff
@@ -179,13 +179,6 @@ describe('DirectDepositCNP', () => {
         view.queryByLabelText(/account number/i),
       );
 
-      // shows a save succeeded alert
```

Comment:
These tests would fail if we properly awaited the result from `findByRole()` since the element it's looking for no longer exists in this component.